### PR TITLE
Mention slack on website

### PR DESCRIPTION
**Pull request by astrofrog**
This replaces gitter by slack on the website. Just to explain the motivation for the change in wording, I feel like previously we made it seem too much like people would actually get an immediate reply, but I think we want to make it very clear this is not the case and that people shouldn't get frustrated if we don't reply immediately (when people use IM services they have waaaaay less patience than email lists). I'm open to suggestions though.

### CODE REVIEW

File: help.html [hunk 1]
```diff
@@ -60,19 +60,43 @@
...
-		<li><a href="https://twitter.com/astropy">@astropy on Twitter</a> - Mention @astropy in a tweet with your question. Your tweet may be re-tweeted in order to reach the relevant community. Many tweets get responses within a few hours.</li>
+		<li><a href="https://twitter.com/astropy">@astropy on Twitter</a> - Mention
+		@astropy in a tweet with your question. Your tweet may be re-tweeted in
+		order to reach the relevant community. Many tweets get responses within a
+		few hours.</li>
+
+		<li><a href="https://www.facebook.com/groups/astropython/">Python Users in
+		Astronomy Facebook group</a> - Post questions or start discussions on
+		anything related to Python programming applied to astronomy. This is a
+		closed group. If your request to join is not approved after a couple days,
```
Comment (bsipocz) [inline help.html]: can we somehow say that this is probably the most helpful channel for beginners?

File: contribute.html [hunk 2]
```diff
@@ -80,8 +80,17 @@
 	welcome and encouraged. These should ideally
 	be posted to our <a href="http://astropy.userecho.com/">UserEcho Forum</a>. </li>
 
-	<li>If you have feedback you would prefer to keep private, you can e-mail <a href="mailto:feedback@astropy.org">feedback@astropy.org</a>. This address points to a private mailing list that includes the astropy core developers. If you would like a reply (e.g., an acknowledgement of your comment), please request it.</li>
-	<li>For the extremely impatient, astropy developers often can be found in <a href="https://gitter.im/astropy/astropy">Astropy's gitter channel</a>. Gitter is basically a live web chat, but with features that make it better for discussing code. If you prefer using IRC clients for chat, there is <a href="https://irc.gitter.im/">a gitter to IRC bridge</a>.</li>
+	<li>If you have feedback you would prefer to keep private, you can e-mail <a
```
Comment (eteq) [inline contribute.html]: You know PEP8's 80 line rules don't apply to HTML, right? :trollface:

File: help.html [hunk 3]
```diff
@@ -60,19 +60,43 @@
...
+		contributors are using Slack (which has a live web chat format)
+		for everyday discussions, so please do feel free to drop by if you have
+		questions about using or contributing to astropy. Be aware that while this
+		is a live chat forum, depending on the time of day/year there may
+		not necessarily be someone available to help, so if you don't get an answer
+		here, be sure to try out one of the other methods of communication!.</li>
+
+		<li> <a href="http://groups.google.com/group/astropy-dev">Developer Email
+		List</a> - Start discussion and ask questions about changing Astropy's
+		behavior or adding new features.</li>
```
Comment (eteq) [inline help.html]: Maybe also mention this is where significant announcements for contributors/developers are usually made? Also maybe point out that this is also where *decisions* get made, not just question-asking.

File: contribute.html (hunk 2 above)
Comment (bsipocz) [reply contribute.html]: Oh, but that's because html is ... (CoC) ..., right? :trollface:

Comment (astrofrog) [reply contribute.html]: 1x55na

Comment (astrofrog) [reply contribute.html]: I PEP8'd some fortran recently

Comment (MSeifert04) [reply contribute.html]: 80 line width isn't actually PEP8, if I [understand it correctly it's a punch card thingy](https://softwareengineering.stackexchange.com/a/148678/198787).

File: help.html (hunk 3 above)
Comment (astrofrog) [reply help.html]: Done

File: help.html (hunk 1 above)
Comment (astrofrog) [reply help.html]: Done

**Comment by astrofrog** [comment]
@bsipocz @eteq - please re-review and let me know if we are good to merge!

**Comment by astrofrog** [comment]
Since @taldcroft and @bsipocz have approved, I'll go ahead and merge!

**Pull request merged**